spec/jobs/wild_collection_job_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:filename) { "example_of_wild_collection_data.csv" }
 
   before do
```

Comment:
Sorry for the nit, but because before(:each) rolls back and before(:all) doesn't, it would be really helpful to make it clearer which behavior we're using here. From a quick SO search I *think* `before` defaults to `before(:each)`, but I'm still not 100% sure of any edge cases etc. If we're now always expanding the use of the hook, it would be nice to know for sure how it's behaving without the next person who sees this having to google-fu.

```suggestion
   before(:each) do
```

Reply:
You're right it defaults to `before(:each)`.  And I think that's unnecessary, will make it more clear and change it to `before(:all)`.

Thanks!